### DISCUSSION TITLE
New to egui and trying it out, Any ideas how to achieve the result similar to the picture.

### DISCUSSION THREAD

**Discussion by animus27** [emilk/egui#2836] [Q&A]
![image](https://user-images.githubusercontent.com/115132000/227328824-46386a49-8ad1-4ab1-b98c-88bb8a5e7e4c.png)
I have this mockup, which I made and I am stumped how do I achieve the result or make a gui similar to that because I am very new to all this gui and coding in rust business. this gui will be for windows. Please help!!

**Comment by rminderhoud** [comment]
The strip demo might be useful to you https://github.com/emilk/egui/blob/master/crates/egui_demo_lib/src/demo/strip_demo.rs

![image](https://user-images.githubusercontent.com/1752584/228672960-199ec733-8078-4192-8754-41c9a9f512f1.png)
